docs/source/user/nalu_run/nalu_inp.rst

```diff
@@ -1007,6 +1033,36 @@ Turbulence averaging
    A boolean flag indicating whether the reynolds stress is
    computed. The default value is ``no``.
 
+.. inpfile:: turbulence_averaging.specifications.compute_resolved_stress
+
+   A boolean flag indicating whether the resolved stress is
+   computed. The default value is ``no``. When this option is turned on, the
+   Favre averages of the resolved quantities are computed as well.
+   
+.. inpfile:: turbulence_averaging.specifications.compute_temperature_resolved_flux
+
+   A boolean flag indicating whether the resolved temperature flux is
+   computed. The default value is ``no``. When this option is turned on, the
+   Favre averages of the resolved temperature is computed as well.
+
+.. inpfile:: turbulence_averaging.specifications.compute_sfs_stress
+
+   A boolean flag indicating whether the sub-filter scale stress is
+   computed. The default value is ``no``. The sub-filter scale stress model is
+   assumed to be of an eddy viscosity type and the turbulent viscosity computed
+   by the turbulence model is used. The sub-filter scale kinetic energy is used
+   to determine the isotropic component of the sub-filter stress. As described
+   in the section :ref:`supp_eqn_set_mom_cons`, the Yoshisawa model is used to
```

Comment:
Is this documentation included in the pull request?

Reply:
Yes. This was in the last commit https://github.com/NaluCFD/Nalu/commit/eb0c2d884d39de857a1cad6e62e97e7d3ec601ff after the discussion during the standup last Thursday.